### DISCUSSION TITLE
In the network campaign game, the bot deconstructs only one item from the stack.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14089] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Although in a single player game the bot normally deconstructs any stack of items by dragging items into the deconstructor one at a time.
But in a network game, the bot only deconstructs the first item from the stack, ignoring the rest. 
I also noticed, in the next round, the remaining items in the stack lose the “deconstruct mark".
It seems like as if deconstruct one item, the mark is removed from the entire stack.

### Reproduction steps

1. create a server
2. start a new campaign game on the Orca sub with one bot.
3. collect 8 oxygen tanks in  your inventory and put them in the closet as one stacks of 8 items.
4. give to the bot "deconstruct" order.
5. Mark  the stack of tanks for deconstruction.
6. The bot will deconstruct one tank, after this the bot will stay inactive.
7. now save and restore the game. The "deconstruct mark" has disappeared from the stack of tanks.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
You have already reported this. https://github.com/FakeFishGames/Barotrauma/discussions/13837

**Reply by pavel4win** [reply]
In the previous version of the game, only one item from the whole stack was marked to deconstruct. And the bot after deconstruct one item have been considered the order completed. This has been fixed in single player in the latest version. In a single-player game, the bot now deconstruct whole stack normally.
Now in a multiplayer game the entire stack is marked for deconstruct , but the bot only deconstructs one item.
The bug's manifestation has changed, so I created a new ticket.
I would like to draw your attention to the fact that in the network game the error has been corrected with an error.

**Reply by Regalis11** [reply]
Ah, sorry, I was too hasty in closing this! I assumed the fix to the previously reported issue hadn't been released yet, but it seems it was included in the hotfix. So there definitely is a bug here still that we need to address.

**Comment by Rokvach** [comment]
Fixed and tested to be working correctly, closing.
